### DISCUSSION TITLE
[BUG] Get-PnPFile

### DISCUSSION THREAD

**Discussion by DXCMSparks** [pnp/powershell#2601] [General]
### Notice
Many bugs reported are actually related to the PnP Framework which is used behind the scenes. Consider carefully where to report an issue:

1. **Are you using ```Invoke-PnPSiteTemplate``` or ```Get-PnPSiteTemplate```**? The issue is most likely related to the Provisioning Engine. The Provisioning engine is _not_ located in the PowerShell repo. Please report the issue here: https://github.com/pnp/pnpframework/issues.
1. **Is the issue related to the cmdlet itself, its parameters, the syntax, or do you suspect it is the code of the cmdlet that is causing the issue?** Then please continue reporting the issue in this repo.
1. **If you think that the functionality might be related to the underlying libraries that the cmdlet is calling** (We realize that might be difficult to determine), please first double check the code of the cmdlet, which can be found here: https://github.com/pnp/powershell/tree/master/src/Commands. If related to the cmdlet, continue reporting the issue here, otherwise report the issue at https://github.com/pnp/pnpframework/issues

### Reporting an Issue or Missing Feature
Get-PnPFile works in foreground but when triggered from a scheduled task getting error 

### Expected behavior 
the file download runs fine when ran from the script not in the task scheduler

### Actual behavior 
Get-PnPFile : The given path's format is not supported .
At C:\#PATHWITHHELD
Get-PnPFile -Url "Shared Documents/General/Sample.xlsb"

### Steps to reproduce behavior
Please include complete script or code samples in-line or linked from [gists](https://gist.github.com/)
Create a scheduled task under the user's account - run while logged in
$C = Connect-PnPOnline -Url YOURURLHERE -UseWebLogin
Get-PnPFile -Url "Shared Documents/General/Sample.xlsb" -Path C:\Temp -FileName Sample.xlsb -Force -Connection $C

### What is the version of the Cmdlet module you are running?
(you can retrieve this by executing ```Get-Module -Name "PnP.PowerShell" -ListAvailable```)
 1.12.0 
### Which operating system/environment are you running PnP PowerShell on?
- [X ] Windows

**Comment by gautamdsheth** [comment]
As you mentioned in the issue, the script works when you run outside the scheduler.
Am moving this to discussion since the issue is not with the module itself but something related to your environment / scheduler.

**Comment by DXCMSparks** [comment]
Interesting enough, If I use the get items from list option it is able to pull the files. I am wondering what the difference is.
